nginx.conf

```
@@ -76,6 +76,7 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/subgraph-debug-forking/$             $scheme://$http_host/docs/$1/subgraphs/cookbook/subgraph-debug-forking/             permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/subgraph-uncrashable/$               $scheme://$http_host/docs/$1/subgraphs/cookbook/subgraph-uncrashable/               permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/substreams-powered-subgraphs/$       $scheme://$http_host/docs/$1/subgraphs/cookbook/substreams-powered-subgraphs/       permanent;
```

Should also update this one to prevent 2 redirects in a row:
```suggestion
        rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/substreams-powered-subgraphs/$       $scheme://$http_host/docs/$1/sps/introduction/       permanent;
```